x/denommetadata/module.go

```diff
@@ -0,0 +1,153 @@
+package denommetadata
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
```

Comment:
not sure this file is needed if you get rid of the denommetadat keeper like I suggested in previosu comment

Reply:
without a module if fails to resolve the URL for the tx